### DISCUSSION TITLE
The page returns before it finishes loading.

### DISCUSSION THREAD

**Discussion by coolsdzy** [FlareSolverr/FlareSolverr#775] [Q&A]
When the session is not used, everything is ok, but it takes a long time, about 30 seconds to solve the challenge. So I created a session to speed up the second opening time. Indeed, the second request returns very quickly because there is no need for a challenge (the returned message is: Challenge not detected!) However, the page returned at this time is incomplete, and the page uses lazy loading, so many contents are is empty, what should I do in this situation?

**Comment by ngosang** [comment]
1. Avoid using sessions
2. Try to open another webpage like www.google.com before opening yours (2 requests to FlareSolverr). This will force Chrome to reload your page completely.

**Comment by mosfetti** [comment]
did anyone solve this?
